src/envo/envo-edit.owl

```diff
@@ -31615,76 +31633,133 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001406> ObjectSomeValuesFrom(<
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001409> "altocumulus cloud"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001409> <http://purl.obolibrary.org/obo/ENVO_01001428>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001409> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_03520001>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001409> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0002124>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001409> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> <http://purl.obolibrary.org/obo/ENVO_00002006>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001409> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000027> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002351> <http://purl.obolibrary.org/obo/ENVO_01000833>))))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_01001410> (altostratus cloud)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-altostratus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001410> "A cloud which is 1) greyish or bluish in colour, 2) organised as a sheet or layer of striated (grooves or channels in cloud formations, arranged parallel to the flow of the air), fibrous or uniform appearance, and 3) having parts thin enough to reveal the local star, at least vaguely, as through ground glass or frosted glass.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-altostratus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001410> "A cloud which 1) constitutes a striated, fibrous, or uniform sheet or layer, 2) is composed of water droplets and ice crystals, 3) is located in the middle level of the atmosphere (middle étage)."@en)
```

Comment:
"constitutes" see comment above, try "occurs in the form of" or similar

Reply:
As above, I've changed it to say "occurs in the form of" instead of "constitutes"

Reply:
Also, I noticed that the `'composed primarily of' some 'object aggregate' ...` axiom was off, and fixed that.